bigquery_etl/glean_usage/app_name_ping_views.py

```diff
@@ -0,0 +1,94 @@
+"""
```

Comment:
We could also name the file something like `glean_app_ping.py` or `glean_app_ping_views.py`